src/components/shared/listItem/ListItem.tsx

```diff
@@ -22,11 +22,15 @@ export function ListItem({ title, text, headingLevel, blueListItem }: ListItemPr
                 {title}
             </Heading>
             {Array.isArray(text) ? (
-                text.map((it) => (
-                    <BodyShort size="small" className={styles.bodyShort} key={it}>
-                        {it}
-                    </BodyShort>
-                ))
+                <ul>
+                    {text.map((it) => (
+                        <li key={it}>
+                            <BodyShort size="small" className={styles.bodyShort}>
```

Comment:
Bare en tanke, men hadde dette funket for oss?

```suggestion
                            <BodyShort key={it} as="li" size="small" className={styles.bodyShort}>
```

Reply:
Det går ann her ja, lurt!